less/components/nav.less

```diff
@@ -23,6 +23,20 @@
         opacity: 1;
       }
     }
+    .nav-lang-selector {
+      position: absolute;
```

Comment:
You can accomplish a right align inside a flex box using this `margin-left: auto;` It's looks pretty arbitrary.